diff --git a/node_modules/react-native-dropdown-picker/src/components/Picker.js b/node_modules/react-native-dropdown-picker/src/components/Picker.js
index e3fb93b..ea9af71 100644
--- a/node_modules/react-native-dropdown-picker/src/components/Picker.js
+++ b/node_modules/react-native-dropdown-picker/src/components/Picker.js
@@ -14,16 +14,15 @@ import {
     TouchableOpacity,
     Text,
     Image,
-    FlatList,
     TextInput,
     Dimensions,
-    ScrollView,
     Modal,
     ActivityIndicator,
     BackHandler,
     Platform,
     StyleSheet,
 } from 'react-native';
+import { FlatList, ScrollView } from 'react-native-gesture-handler';
 
 const { height: WINDOW_HEIGHT } = Dimensions.get('window');
 
@@ -132,7 +131,7 @@ function Picker({
     props = {},
     itemProps = {},
     itemLabelProps = {},
-    badgeProps= {},
+    badgeProps = {},
     modalProps = {},
     flatListProps = {},
     scrollViewProps = {},
@@ -141,18 +140,18 @@ function Picker({
     min = null,
     max = null,
     addCustomItem = false,
-    setOpen = () => {},
-    setItems = () => {},
+    setOpen = () => { },
+    setItems = () => { },
     disableBorderRadius = true,
     containerProps = {},
-    onLayout = (e) => {},
-    onPress = (open) => {},
-    onOpen = () => {},
-    onClose = () => {},
-    setValue = (callback) => {},
-    onChangeValue = (value) => {},
-    onChangeSearchText = (text) => {},
-    onDirectionChanged = (direction) => {},
+    onLayout = (e) => { },
+    onPress = (open) => { },
+    onOpen = () => { },
+    onClose = () => { },
+    setValue = (callback) => { },
+    onChangeValue = (value) => { },
+    onChangeSearchText = (text) => { },
+    onDirectionChanged = (direction) => { },
     zIndex = 5000,
     zIndexInverse = 6000,
     rtl = false,
@@ -162,7 +161,7 @@ function Picker({
     testID,
     closeOnBackPressed = false,
     extendableBadgeContainer = false,
-    onSelectItem = (item) => {}
+    onSelectItem = (item) => { }
 }) {
     const [necessaryItems, setNecessaryItems] = useState([]);
     const [searchText, setSearchText] = useState('');
@@ -187,7 +186,7 @@ function Picker({
      * The item schema.
      * @returns {object}
      */
-     const _schema = useMemo(() => ({...SCHEMA, ...schema}), [schema]);
+    const _schema = useMemo(() => ({ ...SCHEMA, ...schema }), [schema]);
 
     /**
      * componentDidMount.
@@ -235,7 +234,7 @@ function Picker({
                 const _item = items.find(x => x[_schema.value] === item[_schema.value]);
 
                 if (_item) {
-                    return {...item, ..._item};
+                    return { ...item, ..._item };
                 }
 
                 return item;
@@ -303,7 +302,7 @@ function Picker({
     /**
      * Update items in the memory.
      */
-     useEffect(() => {
+    useEffect(() => {
         memoryRef.current.items = necessaryItems;
     }, [necessaryItems]);
 
@@ -352,7 +351,7 @@ function Picker({
      * onPressToggle.
      */
     const onPressToggle = useCallback(() => {
-        const isOpen = ! open;
+        const isOpen = !open;
 
         setOpen(isOpen);
         setSearchText('');
@@ -419,7 +418,7 @@ function Picker({
     /**
      * onScrollToIndexFailed.
      */
-     const onScrollToIndexFailed = useCallback(({averageItemLength, index}) => {
+    const onScrollToIndexFailed = useCallback(({ averageItemLength, index }) => {
         flatListRef.current.scrollToOffset?.({
             offset: averageItemLength * index,
             animated: true
@@ -430,7 +429,7 @@ function Picker({
      * The indices of all parent items.
      * @returns {object}
      */
-     const stickyHeaderIndices = useMemo(() => {
+    const stickyHeaderIndices = useMemo(() => {
         const stickyHeaderIndices = [];
         if (stickyHeader) {
             const parents = sortedItems.filter(item => item[_schema.parent] === undefined || item[_schema.parent] === null);
@@ -514,7 +513,7 @@ function Picker({
      * @returns {object}
      */
     const selectedItems = useMemo(() => {
-        if (! multiple)
+        if (!multiple)
             return [];
 
         return necessaryItems.filter(item => _value.includes(item[_schema.value]));
@@ -548,7 +547,7 @@ function Picker({
      * The multiple text.
      * @returns {string}
      */
-     const _multipleText = useMemo(() => multipleText ?? _('SELECTED_ITEMS_COUNT_TEXT'), [multipleText, _]);
+    const _multipleText = useMemo(() => multipleText ?? _('SELECTED_ITEMS_COUNT_TEXT'), [multipleText, _]);
 
     /**
      * The mode.
@@ -566,12 +565,12 @@ function Picker({
      * Indicates whether the value is null.
      * @returns {boolean}
      */
-     const isNull = useMemo(() => {
+    const isNull = useMemo(() => {
         if (_value === null || (Array.isArray(_value) && _value.length === 0))
             return true;
 
         return necessaryItems.length === 0;
-     }, [necessaryItems, _value]);
+    }, [necessaryItems, _value]);
 
     /**
      * Get the selected item.
@@ -644,7 +643,7 @@ function Picker({
      * onPress.
      */
     const __onPress = useCallback(async () => {
-        const isOpen = ! open;
+        const isOpen = !open;
 
         onPress(isOpen);
 
@@ -676,7 +675,7 @@ function Picker({
      * onLayout.
      */
     const __onLayout = useCallback((e) => {
-        if(Platform.OS !== "web")
+        if (Platform.OS !== "web")
             e.persist();
 
         onLayout(e);
@@ -709,7 +708,7 @@ function Picker({
      * Disable borderRadius for the drop down.
      * @returns {object}
      */
-     const dropDownNoBorderRadius = useMemo(() => {
+    const dropDownNoBorderRadius = useMemo(() => {
         if (listMode === LIST_MODE.MODAL)
             return null;
 
@@ -722,7 +721,7 @@ function Picker({
                 borderBottomRightRadius: 0,
             };
         }
-     }, [disableBorderRadius, open, direction, listMode]);
+    }, [disableBorderRadius, open, direction, listMode]);
 
     /**
      * The disabled style.
@@ -770,7 +769,7 @@ function Picker({
     const _labelStyle = useMemo(() => ([
         THEME.label,
         ...[textStyle].flat(),
-        ...[! isNull && labelStyle].flat(),
+        ...[!isNull && labelStyle].flat(),
         ...[_placeholderStyle].flat(),
     ]), [textStyle, _placeholderStyle, labelStyle, isNull, THEME]);
 
@@ -809,7 +808,7 @@ function Picker({
      * The modal content container style.
      * @returns {object}
      */
-     const _modalContentContainerStyle = useMemo(() => ([
+    const _modalContentContainerStyle = useMemo(() => ([
         THEME.modalContentContainer,
         ...[modalContentContainerStyle].flat()
     ]), [modalContentContainerStyle, THEME]);
@@ -846,13 +845,13 @@ function Picker({
      * @returns {JSX.Element}
      */
     const _ArrowComponent = useMemo(() => {
-        if (! showArrowIcon)
+        if (!showArrowIcon)
             return null;
 
         let Component;
         if (open && ArrowUpIconComponent !== null)
             Component = <ArrowUpIconComponent style={_arrowIconStyle} />;
-        else if (! open && ArrowDownIconComponent !== null)
+        else if (!open && ArrowDownIconComponent !== null)
             Component = <ArrowDownIconComponent style={_arrowIconStyle} />;
         else
             Component = <Image source={open ? ICON.ARROW_UP : ICON.ARROW_DOWN} style={_arrowIconStyle} />;
@@ -876,7 +875,7 @@ function Picker({
      * The icon container style.
      * @returns {object}
      */
-     const _iconContainerStyle = useMemo(() => ([
+    const _iconContainerStyle = useMemo(() => ([
         RTL_STYLE(rtl, THEME.iconContainer),
         ...[iconContainerStyle].flat()
     ]), [rtl, iconContainerStyle, THEME]);
@@ -885,7 +884,7 @@ function Picker({
      * The selected item icon component.
      * @returns {JSX.Element|null}
      */
-     const SelectedItemIconComponent = useMemo(() => {
+    const SelectedItemIconComponent = useMemo(() => {
         const Component = _selectedItemIcon();
 
         if (hideSelectedItemIcon)
@@ -896,7 +895,7 @@ function Picker({
                 <Component />
             </View>
         );
-     }, [_selectedItemIcon, hideSelectedItemIcon, _iconContainerStyle]);
+    }, [_selectedItemIcon, hideSelectedItemIcon, _iconContainerStyle]);
 
     /**
      * The simple body component.
@@ -918,7 +917,7 @@ function Picker({
         setValue(state => {
             let _state = [...state];
             const index = _state.findIndex(item => item === value);
-                _state.splice(index, 1);
+            _state.splice(index, 1);
 
             return _state;
         });
@@ -951,7 +950,7 @@ function Picker({
      * @param {string} str
      * @returns {string}
      */
-     const getBadgeColor = useCallback((str) => {
+    const getBadgeColor = useCallback((str) => {
         str = `${str}`;
 
         const index = Math.abs(ASCII_CODE(str)) % _badgeColors.length;
@@ -963,7 +962,7 @@ function Picker({
      * @param {string} str
      * @returns {string}
      */
-     const getBadgeDotColor = useCallback((str) => {
+    const getBadgeDotColor = useCallback((str) => {
         str = `${str}`;
 
         const index = Math.abs(ASCII_CODE(str)) % _badgeDotColors.length;
@@ -982,7 +981,7 @@ function Picker({
      * Render badge.
      * @returns {JSX.Element}
      */
-    const __renderBadge = useCallback(({item}) => (
+    const __renderBadge = useCallback(({ item }) => (
         <RenderBadgeComponent
             props={badgeProps}
             rtl={rtl}
@@ -1056,7 +1055,7 @@ function Picker({
     const labelContainerStyle = useMemo(() => ([
         THEME.labelContainer, rtl && {
             transform: [
-                {scaleX: -1}
+                { scaleX: -1 }
             ]
         }
     ]), [rtl, THEME]);
@@ -1103,7 +1102,7 @@ function Picker({
      * Extendable badge container.
      * @returns {JSX.Element}
      */
-    const ExtendableBadgeContainer = useCallback(({selectedItems}) => {
+    const ExtendableBadgeContainer = useCallback(({ selectedItems }) => {
         if (selectedItems.length > 0) {
             return (
                 <View style={extendableBadgeContainerStyle}>
@@ -1123,7 +1122,7 @@ function Picker({
      * The badge body component.
      * @returns {JSX.Element}
      */
-     const BadgeBodyComponent = useMemo(() => {
+    const BadgeBodyComponent = useMemo(() => {
         if (extendableBadgeContainer) {
             return <ExtendableBadgeContainer selectedItems={selectedItems} />
         }
@@ -1173,7 +1172,7 @@ function Picker({
     const _listItemContainerStyle = useMemo(() => ([
         RTL_DIRECTION(rtl, THEME.listItemContainer),
         ...[listItemContainerStyle].flat(),
-        stickyHeader && {backgroundColor: THEME.style.backgroundColor},
+        stickyHeader && { backgroundColor: THEME.style.backgroundColor },
     ]), [rtl, listItemContainerStyle, THEME]);
 
     /**
@@ -1183,7 +1182,7 @@ function Picker({
     const _tickIconContainerStyle = useMemo(() => ([
         RTL_STYLE(rtl, THEME.tickIconContainer),
         ...[tickIconContainerStyle].flat()
-    ]), [rtl, tickIconContainerStyle,THEME]);
+    ]), [rtl, tickIconContainerStyle, THEME]);
 
     /**
      * The list item label style.
@@ -1210,7 +1209,7 @@ function Picker({
      */
     const _searchContainerStyle = useMemo(() => ([
         RTL_DIRECTION(rtl, THEME.searchContainer),
-        ...[searchContainerStyle].flat(), ! searchable && ! modalTitle && listMode === LIST_MODE.MODAL && {
+        ...[searchContainerStyle].flat(), !searchable && !modalTitle && listMode === LIST_MODE.MODAL && {
             flexDirection: 'row-reverse'
         }
     ]), [rtl, listMode, searchable, modalTitle, searchContainerStyle, THEME]);
@@ -1289,7 +1288,7 @@ function Picker({
 
         setValue(state => {
             if (multiple) {
-                let _state = state !== null && state !== undefined  ? [...state] : [];
+                let _state = state !== null && state !== undefined ? [...state] : [];
 
                 if (_state.includes(item[_schema.value])) {
                     // Remove the value
@@ -1343,7 +1342,7 @@ function Picker({
             }
         });
 
-        if (closeAfterSelecting && ! multiple)
+        if (closeAfterSelecting && !multiple)
             onPressClose();
     }, [
         setValue,
@@ -1363,7 +1362,7 @@ function Picker({
      * @returns {JSX.Element}
      */
     const _TickIconComponent = useCallback(() => {
-        if (! showTickIcon)
+        if (!showTickIcon)
             return null;
 
         let Component;
@@ -1427,7 +1426,7 @@ function Picker({
      * Render list item.
      * @returns {JSX.Element}
      */
-    const __renderListItem = useCallback(({item}) => {
+    const __renderListItem = useCallback(({ item }) => {
         let IconComponent = item[_schema.icon] ?? null;
 
         if (IconComponent) {
@@ -1525,7 +1524,7 @@ function Picker({
      * @returns {JSX.Element|null}
      */
     const ItemSeparatorComponent = useCallback(() => {
-        if (! itemSeparator)
+        if (!itemSeparator)
             return null;
 
         return (
@@ -1540,7 +1539,7 @@ function Picker({
      * The search placeholder.
      * @returns {string}
      */
-     const _searchPlaceholder = useMemo(() => {
+    const _searchPlaceholder = useMemo(() => {
         if (searchPlaceholder !== null)
             return searchPlaceholder;
 
@@ -1593,7 +1592,7 @@ function Picker({
      * modalTitleStyle.
      * @returns {object}
      */
-     const _modalTitleStyle = useMemo(() => ([
+    const _modalTitleStyle = useMemo(() => ([
         THEME.modalTitle,
         ...[modalTitleStyle].flat(),
         ...[textStyle].flat(),
@@ -1744,7 +1743,7 @@ function Picker({
                     return (
                         <Fragment key={item[_itemKey]}>
                             {index > 0 && ItemSeparatorComponent()}
-                            {__renderListItem({item})}
+                            {__renderListItem({ item })}
                         </Fragment>
                     );
                 })}
